_data/projects/vscode.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Visual Studio Code
+desc: Code editing. Redefined.
+site: https://code.visualstudio.com/
+tags:
+- vscode
+- visual
+- studio
+- code
+- editor
+- ide
+upforgrabs:
+  name: help wanted
+  link: https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3A"help+wanted"
```

Comment:
@joaomoreno Please change this to https://github.com/microsoft/vscode/labels/help%20wanted